### DISCUSSION TITLE
Release v1.12.5 / 5.67.5

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3509] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

The latest update brings notable improvements and fixes to Sandboxie-Plus. 
Key enhancements include the integration of Date & Time display in the Sbie Messages tab, and a streamlined the box creation process, allowing for more versatile box types. Among the various fixes, issues with sandboxed processes and stability concerns in SandMan-v1.12.3 have been addressed, ensuring a more reliable and efficient user experience.

We also announce a change in our update policy: automated update download & installation now requires an active supporter certificate to use the stable channel. Users on the preview channel with all the experimental potentially buggy test builds can still use auto update without a certificate. Users on the stable channel from now on will instead receive a update notification guiding them to our manually download page. 

For a full list of changes and fixes please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1125--5675---2023-12-19).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.12.5'>Release v1.12.5 / 5.67.5</a>.</em>

**Comment by BilleBarrett** [comment]
Cna I get some direction on this error
chrome.exe: SBIE2321 Cannot manage device map:  [C0000034 / 11]
It only happen on one client, and I'd like to be able to fix it 
She does have one mapped drive B: to here backup drive on the Router
Thank you
Bill e. Barrett
PS Happy New Year all

**Reply by isaak654** [reply]
Does it work with the line `NormalFilePath=\Device\Mup` to set in the sandbox?
If not, I would need to know if she is running SbieCtrl/SandMan with administrator rights.

> PS Happy New Year all

You too.

**Comment by BilleBarrett** [comment]
I will look when I get back home, New years things to do 

 

-----------------------------------------------------------------------------------------------------------------


The bridge and the wall are made with the same material

The Bridge joins the people and the Wall divides the people

Choose the Right One

Remote Desktop Services

(336) 988-0891    
 ***@***.***> ***@***.***
